tools/ci/docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+CACHE_DIR=$HOME/gem_cache ./docs/build_docs.sh -p &
+
+for i in `seq 1 30`;
+do
+	echo "Waiting for server..."
+	curl -Is http://localhost:4000 --fail
+	if [ $? -eq 0 ]; then
+		break
+	fi
+	sleep 10
+done
+
+./docs/check_links.sh
```

Comment:
nit: We've made an effort to make the tool scripts in StateFun to be oblivious of the directory they're being executed from (You can also check out the release scripts under `tools/releasing` for a quick copy-paste for this). Could you also do that for this script? Or at least a sanity check that this script is being executed from the project root.

Reply:
Sounds good.